proto/api/v1/image_service.proto

```diff
@@ -34,9 +34,12 @@ message ScanImageRequest {
 }
 
 message ScanImageInternalRequest {
-    storage.ContainerImage image = 1;
-    reserved                       2;
-    bool cached_only             = 3;
+    storage.ContainerImage image              = 1;
+    reserved 2;
+    bool                   cached_only        = 3;
+    string                 cluster_id         = 4;
+    string                 namespace          = 5;
+    repeated string        image_pull_secrets = 6;
```

Comment:
Move this to a sub-message `scope` or `source` or so? Tbh I found it very confusing that in `service_impl.go` we check for `GetClusterId() != ""` and then assume that `GetNamespace()` and `GetImagePullSecrets()` can be used, too. This would be much clearer if we used `if scope := request.GetScope(); scope != nil` and then just `scope.GetClusterId()` etc. This way, we wouldn't have to abuse `cluster_id` as a sentinel in the logic, as it is not more special than the other fields.